extension/package.json

```diff
@@ -332,6 +332,11 @@
         "command": "gradle.createProjectAdvanced",
         "category": "Gradle",
         "title": "Create a Gradle Java Project... (Advanced)"
+      },
+      {
+        "command": "gradle.showGradleProjectsView",
+        "category": "Gradle",
+        "title": "Focus on Gradle Projects View"
```

Comment:
Since we have a specific context here, so it would be better if we can reveal the corresponding node in the Gradle view.